content/docs/rfcs/23/README.md

```diff
@@ -3,47 +3,65 @@ slug: 23
 title: 23/WAKU2-TOPICS
 name: Waku v2 Topic Usage Recommendations
 status: draft
+category: Informational
 editor: Oskar Thoren <oskar@status.im>
 contributors:
   - Hanno Cornelius <hanno@status.im>
+  - Daniel Kaiser <danielkaiser@status.im>
 ---
 
-This document outlines recommended usage of topics in Waku v2.
+This document outlines recommended usage of topic names in Waku v2.
 In [10/WAKU2 spec](/spec/10) there are two types of topics:
 
 - PubSub topics, used for routing
 - Content topics, used for content-based filtering
 
+
 ## PubSub topics
 
 PubSub topics are used for routing of messages, see [11/WAKU2-RELAY](/spec/11) spec for more details of how this routing works.
 
-As written in [10/WAKU2 spec](/spec/10) there is a default PubSub topic:
+[51/WAKU2-RELAY-SHARDING](/spec/51) specifies Relay sharding,
+which allows sharding Waku Relay into a hierarchical set of shards.
+This document (23/WAKU2-TOPICS) comprises recommendations for naming pubsub topics,
+and acts as an informational source for *named sharding*.
+Named sharding is one of the sharding strategies specified in [51/WAKU2-RELAY-SHARDING](/spec/51).
+
+The Waku v2 default PubSub topic is:
 
-`/waku/2/default-waku/proto`
+`/waku/2/default-waku/`
```

Comment:
I'm not sure we need to change the default named pubsub topic (this would require a finicky migration). We can just mention that for legacy reasons it has the `/proto` encoding trailer, but that we no longer support semantic interpretation of that suffix. In other words, it still adheres to the format `/waku/2/{topic-name}`, but for legacy reasons `{topic-name}` is `default-waku/proto`. The same goes for the recently introduced `/waku/2/dev-waku/proto` topic.

Reply:
Agree with @jm-clius. For compatibility with the existing implementations, we should support both topics as the default topic:

- `/waku/2/default-waku/proto`
- `/waku/2/default-waku`

The same way Protocol labs does with the multiaddr protocol `ipfs`, which is an alternate form for `p2p` for backward compatibility reasons.